### DISCUSSION TITLE
Can we hide skipped repos?

### DISCUSSION THREAD

**Discussion by ezeholz** [lowlighter/metrics#451] [Q&A] [answered]
I don't want that everyone can see which repo I'm hiding if it's private. I'm using the same method as token to hide it in action.yml, but it's shown every time that runs.

**Comment by lowlighter** [comment, marked as answer]
Hello 🙂 

If the secret matches exactly the output, GitHub should be able to censor it I think
Does your secrets looks like `repo1,repo2` rather than `repo1, repo2` by any chance?

If yes, then you should try to add a space after the comma, it may solve your issue

**Reply by ezeholz** [reply]
Actually, mine looks like `repo1, repo2`
I'm talking about the chart with the details of every plugin and stuff.
![image](https://user-images.githubusercontent.com/30635862/128254317-6ebd30df-05bd-4c4f-8443-a345f60a00ef.png)
Even when:
![image](https://user-images.githubusercontent.com/30635862/128254411-213b57d0-23c2-4366-9f88-399ff333eb9e.png)

**Reply by ezeholz** [reply]
Nvm, You were right. I believed that Github didn't censor it by default Thanks :D
